dev/Deployment/DeploymentActivityContext.h

```diff
@@ -79,14 +78,9 @@ namespace WindowsAppRuntime::Deployment::Activity
             return m_lastFailure;
         }
 
-        const bool& GetIsMicrosoftPublisherPackage()
+        const bool& GetIsFullTrustPackage()
```

Comment:
NIT: `const bool`

No point being a reference

Also, the method should be const and thus

`const bool GetIsFullTrustPackage() const`